File: 0434-number-of-segments-in-a-string/0434-number-of-segments-in-a-string.java
class Solution {
    public int countSegments(String s) {
        if(s.length() == 0){
            return 0;
        }
        int count = 0;
        List<String> myList = List.of(s.split(" "));
        for (String j : myList){
            if(j != ""){
                count++;
            }
        }
        return count;
    }
}

File: 0434-number-of-segments-in-a-string/0434-number-of-segments-in-a-string.py
class Solution(object):
    def countSegments(self, s):
        if(len(s) == 0):
            return 0
        count = 0
        my_list = s.split(" ")
        for j in my_list:
            if(j != ''):
                count += 1
        return count
        